Readme.md

```diff
@@ -16,6 +16,44 @@ To compile the extension (see instructions below):
 
 ## Installation ##
 
+### Precompiled OS Packages ###
+
+You can install the promscale extension from extension version 0.3.0 with precompiled deb and rpm packages for Debian and RedHat-based distributions. The packages are available with the GitHub [release](https://github.com/timescale/promscale_extension/releases). The extension builds on Postgres' official packages for each distribution and is compatible with Postgres versions 12, 13, and 14.
+
+#### Debian Derivatives ####
+
+1. Install Postgres
+    ```
+    apt-get install -y wget gnupg2 lsb-release
+    echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list
+    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+    apt-get update
+    apt-get install -y postgresql-14
+    ```
+
+2. Install the extension
+    ```
+    wget https://github.com/timescale/promscale_extension/releases/download/0.3.0/promscale_extension-0.3.0.pg14.debian11.x86_64.deb
```

Comment:
**Suggestion:** We probably want to rename the packages I uploaded to GitHub to remove the distro name and version, since it is currently irrelevant and may lead to confusion (users thinking they can't use a `debian11` tagged .deb on `debian10`, etc.). If we end up needing to produce flavored packages in the future, we can add it back. As mentioned in standup, from what I've observed so far, all of the Debian-derived distros and RHEL/Fedora-derived distros use the same paths for Postgres, and we have no other dependencies at the moment, so a single .deb/.rpm for each major version of Postgres is all we need for now.

Reply:
done.